test/src/main/java/org/apache/accumulo/test/CloseScannerIT.java

```diff
@@ -61,14 +68,26 @@ public void testManyScans() throws Exception {
         } // when the scanner is closed, all open sessions should be closed
       }
 
-      List<String> tservers = client.instanceOperations().getTabletServers();
-      int activeScans = 0;
-      for (String tserver : tservers) {
-        activeScans += client.instanceOperations().getActiveScans(tserver).size();
-      }
+      Wait.waitFor(() -> {
+        final List<ActiveScan> activeScans = getAllActiveScans(client);
+        if (activeScans.size() < 3) {
```

Comment:
I would change this to check for 0 as with the wait implemented and retry we should always eventually be at 0.

Reply:
Well my point was that it's not really an error so I don't think it needs to be logged in that level of detail, it's expected since it's closed async so a count seems fine at the debug level since it's expected unless someone else disagrees